### PR TITLE
Fix integration parameters display name in all content

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/30343

## Description
Fix integration parameter display name in all content.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CiscoESAIronPortEmailAPI/ReleaseNotes/1_0_1.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Cisco IronPort EMail API
+- Fixed integration parameters display name.
+- Upgraded the Docker image to demisto/python3:3.9.1.15759.
```
Comment (bakatzir) [inline Packs/CiscoESAIronPortEmailAPI/ReleaseNotes/1_0_1.md:5]:
```suggestion
- Upgraded the Docker image to: *demisto/python3:3.9.1.15759*.
```

File: Packs/CyberX-CentralManager/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### CyberX - Central Manager
+- Fixed integration parameters display name.
+- Upgraded the Docker image to demisto/python3-deb:3.9.1.15758.
```
Comment (bakatzir) [inline Packs/CyberX-CentralManager/ReleaseNotes/1_0_1.md:5]:
```suggestion
- Upgraded the Docker image to demisto/python3-deb:3.9.1.15758.
```
```suggestion
- Upgraded the Docker image to: *demisto/python3-deb:3.9.1.15758*.
```

File: Packs/Cymulate/ReleaseNotes/1_0_8.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Cymulate
+- Fixed integration parameters display name.
+- Upgraded the Docker image to demisto/python3:3.9.1.15759.
```
Comment (bakatzir) [inline Packs/Cymulate/ReleaseNotes/1_0_8.md:5]:
```suggestion
- Upgraded the Docker image to: *demisto/python3:3.9.1.15759*.
```

File: Packs/DeepInstinct/ReleaseNotes/1_0_3.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Deep Instinct
+- Fixed integration parameters display name.
+- Upgraded the Docker image to demisto/python3:3.9.1.15759.
```
Comment (bakatzir) [inline Packs/DeepInstinct/ReleaseNotes/1_0_3.md:5]:
```suggestion
- Upgraded the Docker image to: *demisto/python3:3.9.1.15759*.
```

File: Packs/GitHub/ReleaseNotes/1_1_6.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### GitHub IAM
+- Fixed integration parameters display name.
+- Upgraded the Docker image to demisto/python3:3.9.1.15759.
```
Comment (bakatzir) [inline Packs/GitHub/ReleaseNotes/1_1_6.md:5]:
```suggestion
- Upgraded the Docker image to: *demisto/python3:3.9.1.15759*.
```

File: Packs/Kafka/ReleaseNotes/1_0_2.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Kafka v2
+- Fixed integration parameters display name.
+- Upgraded the Docker image to demisto/pykafka:1.0.0.15212.
```
Comment (bakatzir) [inline Packs/Kafka/ReleaseNotes/1_0_2.md:5]:
```suggestion
- Upgraded the Docker image to: *demisto/pykafka:1.0.0.15212*.
```

File: Packs/PaloAltoNetworks_PAN_OS_EDL_Management/ReleaseNotes/1_0_1.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS EDL Management
+- Fixed integration parameters display name.
+- Upgraded the Docker image to demisto/openssh:1.0.0.12410.
```
Comment (bakatzir) [inline Packs/PaloAltoNetworks_PAN_OS_EDL_Management/ReleaseNotes/1_0_1.md:5]:
```suggestion
- Upgraded the Docker image to: *demisto/openssh:1.0.0.12410*.
```

File: Packs/Snowflake/Integrations/Snowflake/Snowflake.yml [hunk 8]
```diff
@@ -47,7 +47,7 @@ configuration:
   name: isFetch
   required: false
   type: 8
-- display: ┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉‎                                                                                  Fetch query to retrieve new incidents. This field is mandatory when 'Fetches incidents' is set to true.
+- display: Fetch query to retrieve new incidents. This field is mandatory when 'Fetches incidents' is set to true.
```
Comment (bakatzir) [inline Packs/Snowflake/Integrations/Snowflake/Snowflake.yml:50]: nice.

File: Packs/Uptycs/ReleaseNotes/1_0_4.md [hunk 9]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Uptycs
+- Fixed integration parameters display name.
+- Upgraded the Docker image to demisto/uptycs:1.0.0.15503.
```
Comment (bakatzir) [inline Packs/Uptycs/ReleaseNotes/1_0_4.md:5]:
```suggestion
- Upgraded the Docker image to: *demisto/uptycs:1.0.0.15503*.
```

File: Packs/VulnDB/ReleaseNotes/1_0_2.md [hunk 10]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### VulnDB
+- Fixed integration parameters display name.
+- Upgraded the Docker image to demisto/python3:3.9.1.15759.
```
Comment (bakatzir) [inline Packs/VulnDB/ReleaseNotes/1_0_2.md:5]:
```suggestion
- Upgraded the Docker image to: *demisto/python3:3.9.1.15759*.
```

File: Packs/illuminate/ReleaseNotes/1_0_6.md [hunk 11]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Analyst1
+- Fixed integration parameters display name.
+- Upgraded the Docker image to demisto/python3:3.9.1.15759.
```
Comment (bakatzir) [inline Packs/illuminate/ReleaseNotes/1_0_6.md:5]:
```suggestion
- Upgraded the Docker image to: *demisto/python3:3.9.1.15759*.
```

**Review by bakatzir — COMMENTED** [review]
Please reflect the displyname changes in the readme files as well.
@amshamah419 are you ok with the rn note: `Fixed integration parameters display name.`?

File: Packs/CiscoESAIronPortEmailAPI/ReleaseNotes/1_0_1.md (hunk 1 above)
Comment (ChanochShayner) [reply Packs/CiscoESAIronPortEmailAPI/ReleaseNotes/1_0_1.md:5]: done

File: Packs/CyberX-CentralManager/ReleaseNotes/1_0_1.md (hunk 2 above)
Comment (ChanochShayner) [reply Packs/CyberX-CentralManager/ReleaseNotes/1_0_1.md:5]: done

File: Packs/Cymulate/ReleaseNotes/1_0_8.md (hunk 3 above)
Comment (ChanochShayner) [reply Packs/Cymulate/ReleaseNotes/1_0_8.md:5]: done

File: Packs/DeepInstinct/ReleaseNotes/1_0_3.md (hunk 4 above)
Comment (ChanochShayner) [reply Packs/DeepInstinct/ReleaseNotes/1_0_3.md:5]: done

File: Packs/GitHub/ReleaseNotes/1_1_6.md (hunk 5 above)
Comment (ChanochShayner) [reply Packs/GitHub/ReleaseNotes/1_1_6.md:5]: done

File: Packs/Kafka/ReleaseNotes/1_0_2.md (hunk 6 above)
Comment (ChanochShayner) [reply Packs/Kafka/ReleaseNotes/1_0_2.md:5]: done

File: Packs/PaloAltoNetworks_PAN_OS_EDL_Management/ReleaseNotes/1_0_1.md (hunk 7 above)
Comment (ChanochShayner) [reply Packs/PaloAltoNetworks_PAN_OS_EDL_Management/ReleaseNotes/1_0_1.md:5]: done

File: Packs/Uptycs/ReleaseNotes/1_0_4.md (hunk 9 above)
Comment (ChanochShayner) [reply Packs/Uptycs/ReleaseNotes/1_0_4.md:5]: done

File: Packs/VulnDB/ReleaseNotes/1_0_2.md (hunk 10 above)
Comment (ChanochShayner) [reply Packs/VulnDB/ReleaseNotes/1_0_2.md:5]: done

File: Packs/illuminate/ReleaseNotes/1_0_6.md (hunk 11 above)
Comment (ChanochShayner) [reply Packs/illuminate/ReleaseNotes/1_0_6.md:5]: done

**Comment by amshamah419** [comment]
> Please reflect the displyname changes in the readme files as well.
> @amshamah419 are you ok with the rn note: `Fixed integration parameters display name.`?

Let's use: `Fixed display names for various integration parameters.`

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml [hunk 12]
```diff
@@ -14,10 +14,7 @@ configuration:
   defaultvalue: ""
   type: 0
   required: true
-- display: |-
-    ______________ Basic Authentication _____________
-
-    Username
+- display: Username
```
Comment (ShahafBenYakir) [inline Packs/Jira/Integrations/JiraV2/JiraV2.yml:17]:
```suggestion
- display: Username (Basich Authentication)
```
what do you think? @ChanochShayner @amshamah419 @bakatzir

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml [hunk 13]
```diff
@@ -32,10 +29,7 @@ configuration:
   defaultvalue: ""
   type: 4
   required: false
-- display: |-
-    __________________ OAuth 1.0 __________________
-
-    ConsumerKey
+- display: Consumer Key
```
Comment (ShahafBenYakir) [inline Packs/Jira/Integrations/JiraV2/JiraV2.yml:32]:
```suggestion
- display: Consumer Key (OAuth 1.0)
```
what do you think? @ChanochShayner @amshamah419 @bakatzir

File: Packs/Kafka/Integrations/Kafka_V2/Kafka_V2.yml [hunk 14]
```diff
@@ -30,10 +27,7 @@ configuration:
   name: additional_password
   required: false
   type: 4
-- display: |-
-    ┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉
-    ‎                               Fetch Incidents Settings
-    Topic to fetch incidents from (Required for fetch incidents)
+- display: Fetch Incidents Settings Topic to fetch incidents from (Required for fetch incidents)
```
Comment (ShahafBenYakir) [inline Packs/Kafka/Integrations/Kafka_V2/Kafka_V2.yml:30]:
```suggestion
- display: Topic to fetch incidents from (Required for fetch incidents)
```

File: Packs/Kafka/Integrations/Kafka_V2/Kafka_V2.yml [hunk 15]
```diff
@@ -11,10 +11,7 @@ configuration:
   name: use_ssl
   required: false
   type: 8
-- display: |-
-    ┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉
-    ‎                                   Certificate Settings
-    CA certificate of Kafka server (.cer)
+- display: Certificate Settings CA certificate of Kafka server (.cer)
```
Comment (ShahafBenYakir) [inline Packs/Kafka/Integrations/Kafka_V2/Kafka_V2.yml:14]:
```suggestion
- display: CA certificate of Kafka server (.cer)
```

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml (hunk 13 above)
Comment (bakatzir) [inline Packs/Jira/Integrations/JiraV2/JiraV2.yml:32]: maybe add an additionalinfo to explain it is used in oauth1.0?

**Review by ShahafBenYakir — COMMENTED** [review]
@ChanochShayner Nice work. Let's run nightly on this (we touched many integrations here)

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml (hunk 13 above)
Comment (ShahafBenYakir) [reply Packs/Jira/Integrations/JiraV2/JiraV2.yml:32]: @bakatzir Not supported in 5.0 (maybe also 5.5 don't remember)

File: Packs/IntSight/Integrations/IntSight/IntSight.yml [hunk 16]
```diff
@@ -44,7 +44,7 @@ configuration:
   name: incidentType
   required: false
   type: 13
-- display: '(MSSP accounts only) Sub account ID:'
+- display: MSSP sub account ID
```
[the remark this answers is not in the document]
Comment (ChanochShayner) [reply Packs/IntSight/Integrations/IntSight/IntSight.yml:47]: done

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml (hunk 12 above)
Comment (ChanochShayner) [reply Packs/Jira/Integrations/JiraV2/JiraV2.yml:17]: done

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml (hunk 13 above)
Comment (ChanochShayner) [reply Packs/Jira/Integrations/JiraV2/JiraV2.yml:32]: done

File: Packs/Kafka/Integrations/Kafka_V2/Kafka_V2.yml (hunk 14 above)
Comment (ChanochShayner) [reply Packs/Kafka/Integrations/Kafka_V2/Kafka_V2.yml:30]: done

File: Packs/Kafka/Integrations/Kafka_V2/Kafka_V2.yml (hunk 15 above)
Comment (ChanochShayner) [reply Packs/Kafka/Integrations/Kafka_V2/Kafka_V2.yml:14]: done

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**